src/proxy.c

```diff
@@ -31,3 +31,9 @@ int git_proxy_options_dup(git_proxy_options *tgt, const git_proxy_options *src)
 
 	return 0;
 }
+
+void git_proxy_options_free(git_proxy_options *opts)
```

Comment:
We usually call this kind of function `git_proxy_options_clear` instead of `free`. A `free` should also free the structure itself.

Reply:
Sure, that makes sense. Done.